### PR TITLE
Added inshorts.md as a remote company

### PR DESCRIPTION
**Pull request by shivjha-2712**
This Pull Request should adhere to the repository's [Code of Conduct](../../main/.github/CONTRIBUTING.md).

Title your pull request descriptively. `[edit X company]` or `[add new company - COMPANY]` 

Please read and check the boxes in the list below that apply!

If updating or adding a company, please include details here: _delete this and add details_

Check off this list as appropriate (add an `x` between the `[]`). It will be used by @dougaitken when reviewing your contribution.

- [] This PR contains housekeeping only (URL edits, copy changes etc) - **if this is checked, delete other lines that don't apply.**
- [] You are an employee of the company mentioned and confirm all included details are correct.
- [x] You know your alphabet - company is listed in alphabetical order in the README.md file.
- [x] The company directly hires employees. ** No bootcamps / freelance sites / etc**
- [x] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such.
- [x] A [company profile](/company-profiles/example.md) is included - This is __required__ for new additions.
- [x] An entry has been added or edited within the [readme.md file](../../main/README.md).
- [x] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
- [x] __Region__ details the geographic regions in which this company employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
- [x] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

This is a modified version of the [Contributing Guidelines](../../main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by shivjha-2712** [comment]
Could you just help me out with this

File: README.md
Comment (KieranRobson) [inline README.md:325]: This should be below inquicker

File: company-profiles/inshorts.md
Comment (KieranRobson) [inline company-profiles/inshorts.md:3]: This should be renamed to `Company Blurb`

**Comment by dougaitken** [comment]
@shivjha-2712 Please see @KieranRobson's reviews.

Also, you checked "housekeeping only" in the PR template, this isn't just housekeeping.

**Comment by shivjha-2712** [comment]
> @shivjha-2712 Please see @KieranRobson's reviews.
> 
> Also, you checked "housekeeping only" in the PR template, this isn't just housekeeping.

Have made changes as per the Reviewer requested and also have updated the comment and have removed the checkmark of "housekeeping-only" section. I hope all the things are sorted and now it can be merged

**Review by shivjha-2712 — COMMENTED** [review]
Have made the changes please accept and merge it accordingly

**Comment by shivjha-2712** [comment]
I think you would have missed it could you please review it again and merge it for me

**Comment by KieranRobson** [comment]
Please be patient with merges, I cannot merge as I do not have merge rights, and people have lives around the open-source community that they maintain or partake in. If you are worried about hacktoberfest, there is still time!

**Review by dougaitken — APPROVED** [review]

**Comment by dougaitken** [comment]
Hey @shivjha-2712, thanks for contributing, it is appreciated.

I'm aware of every change in this repository. Your PR will be merged in soon.

**Pull request merged**
